go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/hashicorp/packer-plugin-vsphere
 
-go 1.21.0
+go 1.22
+
+toolchain go1.23.2
```

Comment:
I would suggest leaving this out of the `go.mod` file; also the `go` directive should now be a `v<major>.<minor>.<patch>` starting with 1.22.0, so I suggest we fix this before merging.

Reply:
Resolved in https://github.com/hashicorp/packer-plugin-vsphere/commit/80f6f37135819f0211f210761f46108b7d264faa.